XamlStyler.Package/StylerPackage.cs

```diff
@@ -186,6 +183,9 @@ private void Execute(Document document)
                 stylerOptions.ConfigPath = configPath;
             }
 
+            stylerOptions.IndentSize = Int32.Parse(xamlEditorProps.Item("IndentSize").Value.ToString());
```

Comment:
Couldn't agree more with you! But then again, that's not my code, I just moved my lines of code from place!